### ISSUE TITLE
How to identify the observed target in the callback.

### ISSUE THREAD

**Issue by rkaule** [ljbc1994/BlazorIntersectionObserver#39]
Hi,
I have a group of dynamic elements that I load on a page. Each of them is contained in an IntersectionObserve where I specify the options and the callback.  It is working perfectly.  However when I am in the callback function I have no way of determining which of the elements the callback is from (other than the bounding box from the entry).  Am I missing something?  I need to call a C# function from the callback and I need to know which of the elements it is for.

Thanks,
Bob

**Comment by ljbc1994** [comment]
Hi @rkaule

Could you use some sort of identifier to achieve this in the loop - for example something like:

```razor
@foreach (var image in Images)
{
    <IntersectionObserve Once=@true>
       <div class="example" @ref="context.Ref.Current" OnChange="@(entry => OnIntersectingChanged(image.Id, entry))">
    </IntersectionObserve>
}

@functions {
    public void OnIntersectingChanged(string id, IntersectionObserverEntry entry)
    {
         
    }
}
```

**Comment by rkaule** [comment]
Hi Louie,
Thanks for the reply.  I can try this.  While I was waiting I downloaded the source.  I added Target : String to the IntersectionObserverEntry class, then in the index.ts I added Target = entry.target.id ( This may not be exactly what I did, I don't have the code in front of me right now, but I think you know where I'm talking about.).  I got this to work, but trying to get the nugget package compiled correctly was a pain.  I use Rider on a Mac.  I will try your suggestion this coming week and let you know.

Thanks,
Bob
